tests/yam/agama/boot_agama.pm

```diff
@@ -25,6 +25,15 @@ use bootloader_s390;
 use bootloader_zkvm;
 use bootloader_pvm;
 
+sub set_agama_version {
+    select_console 'install-shell';
+
+    my $info_file = script_output("cat /var/log/build/info");
+    if ($info_file =~ /^Image.version:\s+(?<major_version>\d+)\./m) {
+        set_var("AGAMA_VERSION", $+{'major_version'});
```

Comment:
a record_info only with the AGAMA_VERSION would be also faster to read when only interested in the value of the variable, could you please add it?